Darwin missing declaration of 'pollfd'

**darwin-poll.zig**:
```zig
const std = @import("std");

test "poll" {
    std.os.poll(&[_]std.os.pollfd{.{
        .fd = std.io.getStdIn().handle,
        .events = std.os.POLLIN,
        .revents = undefined,
    }}, 0);
}
```

**Output**:
```console
$ zig test darwin-poll.zig
/usr/local/Cellar/zig/0.6.0_1/lib/zig/std/os.zig:4148:20: error: use of undeclared identifier 'pollfd'
pub fn poll(fds: []pollfd, timeout: i32) PollError!usize {
                   ^
./darwin-poll.zig:4:11: note: referenced here
    std.os.poll(&[_]std.os.pollfd{.{
          ^
```

For Linux, `pollfd` is defined (along with the other macros and types used by `poll`) in `lib/std/os/bits/linux.zig`, but these are missing in `lib/std/os/bits/darwin.zig`, even though Darwin supports the same `poll` interface as Linux.

--- a/lib/std/os/bits/darwin.zig
+++ b/lib/std/os/bits/darwin.zig
@@ -1430,3 +1430,29 @@ pub const LOCK_SH = 1;
 pub const LOCK_EX = 2;
 pub const LOCK_UN = 8;
 pub const LOCK_NB = 4;
+
+pub const nfds_t = usize;
+pub const pollfd = extern struct {
+    fd: fd_t,
+    events: i16,
+    revents: i16,
+};
+
+pub const POLLIN = 0x001;
+pub const POLLPRI = 0x002;
+pub const POLLOUT = 0x004;
+pub const POLLRDNORM = 0x040;
+pub const POLLWRNORM = POLLOUT;
+pub const POLLRDBAND = 0x080;
+pub const POLLWRBAND = 0x100;
+
+pub const POLLEXTEND = 0x0200;
+pub const POLLATTRIB = 0x0400;
+pub const POLLNLINK = 0x0800;
+pub const POLLWRITE = 0x1000;
+
+pub const POLLERR = 0x008;
+pub const POLLHUP = 0x010;
+pub const POLLNVAL = 0x020;
+
+pub const POLLSTANDARD = POLLIN | POLLPRI | POLLOUT | POLLRDNORM | POLLRDBAND | POLLWRBAND | POLLERR | POLLHUP | POLLNVAL;

